### PR TITLE
Add a robots.txt to s3-proxy

### PR DESCRIPTION
**Pull request by dimaryaz**
Block all indexing in the s3 proxy.

### CODE REVIEW

File: s3-proxy/nginx.conf [hunk 1]
```diff
@@ -69,6 +69,11 @@ http {
             add_header 'Content-Type' 'text/plain';
             return 200 'S3 Proxy';
         }
+
+        location = /robots.txt {
```
Comment (akarve) [inline s3-proxy/nginx.conf:73]: Does this work for head requests? Some bots HEAD before GET, etc.

**Review by akarve — APPROVED** [review]

File: s3-proxy/nginx.conf (hunk 1 above)
Comment (dimaryaz) [reply s3-proxy/nginx.conf:73]: Yep

File: s3-proxy/nginx.conf [hunk 2]
```diff
@@ -69,6 +69,11 @@ http {
...
+            add_header 'Content-Type' 'text/plain';
+            return 200 'User-agent: *\nDisallow: /\n';
```
Comment (akarve) [inline s3-proxy/nginx.conf:75]: I was wondering if we need a terminating `\n`. Most examples I saw don't have it but I doubt it matters.

Comment (dimaryaz) [reply s3-proxy/nginx.conf:75]: Pretty sure it doesn't matter - but in general, files should end with a `\n`

**Pull request merged**